README.md

```diff
@@ -2301,4 +2303,64 @@ Giga  -  1,073,741,824
 Tera  -  1,099,511,627,776
 Peta  -  1,125,899,906,842,624
 Exa   -  1,152,921,504,606,847,000
-```
\ No newline at end of file
+```
+
+## Databases and Indices
+
+MongoDB is the most popular document-based database. It allows several databases which can be used for test, live etc.
+
+MongoDB is made up of collections of documents and fields. All documents must have IDs, but any other fields are optional.
+
+IDs in Mongo are automatically Object IDs. These IDs have a lot of power and contain a lot of information, such as when the document was created. The user can override the generated ID with a specific ID if required.
+
+### Useful MongoDB commands
+
+- `show dbs` - shows all available databases
+- `use` followed by database name - chooses the database the user would like to work on
+- `show collections` - shows the documents
+- `cls` - clears the screen
+- `COLLSCAN` - scans the collections
```

Comment:
`COLLSCAN` and `IXSCAN` are not commands -- they identify the strategy  used for querying documents.